flexvol/flexvoldriver.go

```diff
@@ -230,11 +231,11 @@ func doMount(destinationDir string, ninputs *creds.Credentials, workloadPath str
 		cmd := exec.Command("/bin/unmount", destinationDir)
 		e := cmd.Run()
 		if e != nil {
-			_ = logWriter.Warning(fmt.Sprintf("failed to unmount %s\n", destinationDir))
+			failure("doMount", inp, fmt.Sprintf("failed to unmount %s\n", destinationDir))
```

Comment:
I think this could allow `failure` to be called multiple times, resulting in multiple JSON responses printed to stdout, which does not match the spec.  In the current code these warnings don't result in "failure" of the overall method, so maybe stick with that behavior where we try to log to syslog, but press on regardless of whether we were able to log to syslog.

Reply:
Note this occurs in a couple different places in this changeset

Reply:
Fixed